.openapi/service-accounts.yaml

```diff
@@ -1262,7 +171,7 @@ paths:
   /apis/service_accounts/v1/{id}/resetSecret:
     post:
       tags:
-      - service_accounts
+      - serviceaccounts
```

Comment:
Do you mind changing this on your side (giving hint to generator)
For some of our sdks this will cause compilation errors and bad user api

Reply:
replaced "serviceaccounts" with "service_accounts"  in tags